README.md

```diff
@@ -1,44 +1,19 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Cruzhacks 2020 Website
```

Comment:
I think it would be valuable to have a `tests` directory inside of `src`. Please add that directory and update the documentation in the README.

Reply:
You got it.